### PR TITLE
GLPI: Add format

### PR DESCRIPTION
**Pull request by pferate**
There may be more useful data to pull from the database (opened/closed per time period, number closed by a user, etc.), but I'm not familiar with GLPI, so I'm just keeping the format as-is.

Working towards #98

### CODE REVIEW

**Comment by tobes** [comment]
Yes,  It is much easier if you only change one thing at a time as it makes the reviews much easier.

File: py3status/modules/glpi.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
     user: login user
     warning: set degraded color above this threshold
 
+Format placeholders:
+    {num_open} The number of open tickets
```
Comment (pferate) [inline py3status/modules/glpi.py]: Either of those are fine with me.  `{open}` seems too ambiguous without context.  `{tickets_open}` would be what I would lean towards.

Comment (tobes) [inline py3status/modules/glpi.py]: Let's go with `{tickets_open}`

**Comment by ultrabug** [comment]
@pferate thanks for this, please check the result of the travis CI which shows that the default format should be documented in the docstring as well !

When travis will pass, I'll merge :+1:

**Comment by pferate** [comment]
@ultrabug, I rebased this branch and added the missing docstring entry.  PTAL

**Comment by ultrabug** [comment]
Awesome, thanks @pferate !

**Pull request merged**